pkgs/by-name/te/technitium-dns-server/package.nix

```
@@ -9,13 +9,13 @@
 }:
 buildDotnetModule rec {
   pname = "technitium-dns-server";
-  version = "13.2";
+  version = "13.6";
```

```suggestion
  version = "13.6.0";
```
Because that seems to be the correct tag to use.